docs/backend/sampling_params.md

```diff
@@ -55,6 +55,7 @@ Please refer to our dedicated guide on [constrained decoding](https://docs.sglan
 * `ignore_eos`: Don't stop generation when EOS token is sampled.
 * `skip_special_tokens`: Remove special tokens during decoding.
 * `custom_params`: Used when employing `CustomLogitProcessor`. For usage see below.
+* `return_hidden_states`: Whether to return hidden states of the model. Note that each time it changes, the cuda graph will be recaptured, which might lead to a performance hit. See the [examples](https://github.com/sgl-project/sglang/blob/main/examples/runtime/engine/hidden_states.py) for more information.
```

Comment:
It is not a sampling parameter. Move it here https://github.com/sgl-project/sglang/blob/6b859e7ddd8178823dc500a52cd172791b15677b/python/sglang/srt/managers/io_struct.py#L61

Reply:
Thanks! I will fix it.